src/app.js

```diff
@@ -35,7 +35,7 @@ app.use((req, res, next) => {
   // Request methods you wish to allow
   res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE')
   // Request headers you wish to allow
-  res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type,authorization')
+  res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,Content-Type,Authorization')
```

Comment:
It actually shouldn't be there. It's not an array. It's a CSV string, so spaces are usually not placed.